frontend/src/metabase/reference/reference.js

```diff
@@ -250,30 +191,14 @@ export const rUpdateFieldDetail = (formFields, props) => {
   updateDataWrapper(props, props.updateField)(formFields);
 };
 
-export const rUpdateMetricDetail = async (metric, guide, formFields, props) => {
+export const rUpdateMetricDetail = async (metric, formFields, props) => {
   props.startLoading();
   try {
     const editedFields = filterUntouchedFields(formFields, metric);
     if (!isEmptyObject(editedFields)) {
       const newMetric = { ...metric, ...editedFields };
       await props.updateMetric(newMetric);
-
-      const importantFieldIds = formFields.important_fields.map(
-        field => field.id,
-      );
-      const existingImportantFieldIds =
-        guide.metric_important_fields &&
-        guide.metric_important_fields[metric.id];
-
-      const areFieldIdsIdentitical =
-        existingImportantFieldIds &&
-        existingImportantFieldIds.length === importantFieldIds.length &&
-        existingImportantFieldIds.every(id => importantFieldIds.includes(id));
-
-      if (!areFieldIdsIdentitical) {
-        await props.updateMetricImportantFields(metric.id, importantFieldIds);
-        wrappedFetchMetricDetail(props, metric.id);
-      }
+      wrappedFetchMetricDetail(props, metric.id);
```

Comment:
```suggestion
```
I'm pretty sure we can delete this line. I think it was only needed to fetch more metadata if the important fields changed.